study-builder/tenants/rgp/pages/login.html

```diff
@@ -17,63 +18,337 @@
 <script src="https://cdn.auth0.com/js/es5-shim.min.js"></script>
 <![endif]-->
 
-<script src="https://cdn.auth0.com/js/lock/11.27/lock.min.js"></script>
+<script src="https://cdn.auth0.com/js/lock/11.5.2/lock.min.js"></script>
```

Comment:
Curious why we downgrade the version of auth0 lock library?

Reply:
I think I just copied one of the existing Auth0 login pages to give myself a head start and forgot to check the version.  I'll update the version and retest.